src/Badge/Model/Package.php

```diff
@@ -20,7 +20,7 @@
 class Package
 {
     private static string $modifierRegex = '[._-]?(?:(stable|beta|b|RC|alpha|a|patch|pl|p)(?:[.-]?(\d+))?)?([.-]?dev)?';
-    private string $license;
+    private string $license = '';
     private ApiPackage $originalObject;
     private ?string $latestStableVersion = null;
     private ?string $latestUnstableVersion = null;
```

Comment:
I think we should set up the property `$licence` like any others `$latestStableVersion` and `$latestUnstableVersion`.  
Should we do that we should refactor too much code?

Reply:
Property should be initialized with `null`, since it's nullable (related getter returns `?string`).

Reply:
I have updated with null.